### DISCUSSION TITLE
What table is the OECD income inequality data in?

### DISCUSSION THREAD

**Discussion by MaxGhenis** [owid/etl#2602] [Q&A]
I'm trying to use some of the data in https://ourworldindata.org/income-inequality-before-and-after-taxes, and found the LIS table (`luxembourg_income_study`), but can't find the OECD one. Is this in the catalog? More generally, is there a way to tie an OWID visualization to its table? Thanks!

<img width="661" alt="image" src="https://github.com/owid/etl/assets/6076111/1e9c7341-c3cf-4c09-ba33-07ab80d88396">

<img width="322" alt="image" src="https://github.com/owid/etl/assets/6076111/2678dcbc-a37e-45a7-abc8-875e8c934396">

<img width="425" alt="image" src="https://github.com/owid/etl/assets/6076111/cb5ac982-e1e2-4c39-b6a1-494e6c77899d">

**Comment by lucasrodes** [comment]
Hi @MaxGhenis! Thanks for your interest in our project.

We've been having internal discussions on what's the best way to tie our catalog with our site's charts. Hopefully we can make some progress in the near future, apologies for the confusion!

For the time being, some thoughts addressing your question:

1. The table containing OECD's indicators on income distribution has the ETL path `oecd/2024-04-10/income_distribution_database/income_distribution_database`.
2. This means `oecd` namespace, version `2024-04-10`, dataset name `income_distribution_database` and table name `income_distribution_database`. Read more on these fields in [our docs](https://docs.owid.io/projects/etl/architecture/).
3. When using our python API, I'd recommend using the keyword arguments of the function `find`. Instead of using `catalog.find("oecd")`, use `catalog.find(table="oecd")`.
4. Note that since the table name (mentioned in 2) does not include word "oecd" it will not be shown.
5. In this case, a search by namespace would be better, since we usually store all OECD datasets under the namespace `oecd`. E.g. `catalog.find(namespace="oecd")`. But now that you already know the table name, you can simply run `catalog.find_latest(table="income_distribution_database")` to load the latest version (in this case `2024-04-10`).

**Reply by lucasrodes** [reply]
@MaxGhenis FYI, we are working to expose a new API so that you can get the relevant indicators for a given chart. More details here: https://github.com/owid/etl/pull/2629
